tests/test_functional.py

```diff
@@ -115,7 +115,6 @@ def test_obal_scratch_upstream_hello():
     assert os.path.exists('packages/hello/hello-2.10.tar.gz')
 
     expected_log = [
-        "koji list-pkgs --tag obaltest-nightly-rhel7 --package hello --quiet",
```

Comment:
If one does not exist already, I feel adding a test that configures `build_package_koji_whitelist_check` to true that ensures as part of this command is called would be good since its still an available option.

Reply:
So, I ended up adding two checks, one test for each setting of the variable (true, false), to upstream and downstream sections.

I also added a couple assert functions to assist, ensuring the tests only care if their specific output appears in the log, instead of testing the entire log content.